packages/email/mailer.ts

```diff
@@ -46,10 +46,14 @@ const getTransport = () => {
     host: process.env.NEXT_PRIVATE_SMTP_HOST ?? 'localhost:2500',
     port: Number(process.env.NEXT_PRIVATE_SMTP_PORT) || 587,
     secure: process.env.NEXT_PRIVATE_SMTP_SECURE === 'true',
-    auth: {
-      user: process.env.NEXT_PRIVATE_SMTP_USERNAME ?? '',
-      pass: process.env.NEXT_PRIVATE_SMTP_PASSWORD ?? '',
-    },
+    ignoreTLS: process.env.NEXT_PRIVATE_SMTP_SECURE === 'false' ? true : undefined,
+    auth:
+      process.env.NEXT_PRIVATE_SMTP_USERNAME !== ''
```

Comment:
If the environment variable `NEXT_PRIVATE_SMTP_USERNAME` is not set (i.e., undefined) the condition `process.env.NEXT_PRIVATE_SMTP_USERNAME !== ''` will evaluate to true. I'm suggesting you check for such case, like below

```ts
process.env.NEXT_PRIVATE_SMTP_USERNAME && process.env.NEXT_PRIVATE_SMTP_USERNAME !== ''
```

Reply:
i missed that, thanks for the feedback. Its implemented now.